docs/user/dashboard/lens.asciidoc

```diff
@@ -176,6 +178,29 @@ Compare your real-time data set to the results that are offset by a time increme
 
 For a time shift example, refer to <<compare-time-ranges>>.
 
+[float]
+[[add-reference-lines]]
+==== Add reference lines
+
+With reference lines, you can identify specific values in your visualizations with icons, colors, and other display options. You can add reference lines to any visualization type that displays axes.
```

Comment:
i know that axes is the plural of axis, but I keep thinking of an ax.  could it be something like "displays vertical and horizontal axes"?